README.md

```diff
@@ -229,6 +229,7 @@ Application | GitHub Link | Store/Marketplace
 :-:|:--|:--
 Visual Studio Code | [warpnet/vscode-salt-lint](https://github.com/warpnet/vscode-salt-lint) | [VisualStudio Marketplace](https://marketplace.visualstudio.com/items?itemName=warpnet.salt-lint)
 Sublime Text | [warpnet/SublimeLinter-salt-lint](https://github.com/warpnet/SublimeLinter-salt-lint) | [Package Control](https://packagecontrol.io/packages/SublimeLinter-contrib-salt-lint)
+Vim dense-analysis/ale plugin| [warpnet/SublimeLinter-salt-lint](https://github.com/dense-analysis/ale) | [Package Control](https://github.com/dense-analysis/ale)
```

Comment:
Thanks @Poulpatine for adding **salt-lint** support to the Asynchronous Lint Engine! 🎉 

I've made a slight suggestion for updating the entry in the plugins table:

```suggestion
Vim ([ALE plugin](https://github.com/dense-analysis/ale)) | [dense-analysis/ale](https://github.com/dense-analysis/ale) | [GitHub](https://github.com/dense-analysis/ale)
```